diff --git a/scripts/install_libsharp.sh b/scripts/install_libsharp.sh
index fcfa0b7..eeabb84 100755
--- a/scripts/install_libsharp.sh
+++ b/scripts/install_libsharp.sh
@@ -23,7 +23,7 @@ if [[ $TRAVIS ]] ; then
     CFLAGS="-std=c99 -O3 -ffast-math"
 else
     echo "Using -march=native. Binary will not be portable."
-    CFLAGS="-march=native -std=c99 -O3 -ffast-math"
+    CFLAGS+="-march=native -std=c99 -O3 -ffast-math"
 fi
 CFLAGS=$CFLAGS ./configure --prefix=${PWD}/../ --enable-shared=no --with-pic=yes
 make
